x/wasm/keeper/querier_test.go

```diff
@@ -283,13 +283,13 @@ func TestQueryContractListByCodeOrdering(t *testing.T) {
 	}
 
 	// create 10 contracts with real block/gas setup
-	for i := range [10]int{} {
+	for i := 0; i < 10; i++ {
 		// 3 tx per block, so we ensure both comparisons work
 		if i%3 == 0 {
 			ctx = setBlock(ctx, h)
 			h++
 		}
-		_, _, err = keepers.ContractKeeper.Instantiate(ctx, codeID, creator, nil, initMsgBz, fmt.Sprintf("contract %d", i), topUp)
+		_, _, err = keepers.ContractKeeper.Instantiate(ctx, codeID, creator, nil, initMsgBz, fmt.Sprintf("contractAddr %d", i), topUp)
```

Comment:
Is this diff intended? Why is `i` a contract address?

Reply:
It is a label not an address. good 👁️